source/getting-started/index.markdown

```diff
@@ -41,11 +41,20 @@ Follow this guide if you want to get started with Home Assistant easily, or if y
 
 We will need a few things to get started with installing Home Assistant. The latest Raspberry Pi model makes a good and affordable starting point for your home automation journey. Links below are linking to Amazon US. If you're not in the US, you should be able to find these items in web stores in your country.
 
-- [Raspberry Pi 3 Model B+](https://amzn.to/2IAyNl0) + [Power Supply](https://www.raspberrypi.org/help/faqs/#powerReqs) (at least 2.5A)
+- [Raspberry Pi 4 Model B](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/), 1 GB RAM version is enough, but if you can afford it, 2 or 4 GB RAM won't hurt + [Power Supply](https://www.raspberrypi.org/products/type-c-power-supply/) (at least 3.0A)
 - [Micro SD Card](https://amzn.to/2X0Z2di). Ideally get one that is [Application Class 2](https://www.sdcard.org/developers/overview/application/index.html) as they handle small I/O much more consistently than cards not optimized to host applications. Size 32 GB or bigger recommended.
 - SD Card reader. Part of most laptops, and also available as [standalone USB sticks](https://amzn.to/2WWxntY) (the brand doesn't matter, just pick the cheapest)
 - Ethernet cable (optional, Hass.io can work with WiFi as well)
 
+
+<p class='note'>
+For advanced users:
+ 
+- Alternatively you can invest in small SSD and USB-Sata adapter: for example [Samsung 850 Evo 120GB or more] (https://smile.amazon.com/Samsung-250GB-Internal-MZ-76E250B-AM/dp/B07864WMK8/) and high quality [Usb-Sata adapter] (https://smile.amazon.com/UGREEN-External-Enclosure-Adapter-Housing/dp/B06XWRRMYX)
```

Comment:
As I told you in the previous PR, this does not apply here.
The getting started aims for new users and uses HassOS. You cannot use these for HassOS.

Reply:
> There you go @qqgg231, I've pointed out the same things again.
> You wanted to improve as you mentioned in the previous PR. As the previous PR contained nothing mergable (as in, none of the content changes where viable), creating a new PR for other changes seems way more useful.
> 
> Nevertheless, you created a PR that contained the same flaws as the previous one. 🤷‍♂

@frenck, Why not recommend the newest hardware to users?

Why not change links to "geni.us" to have more money for homeassistant project?

Reply:
> As I told you in the previous PR, this does not apply here.
> The getting started aims for new users and uses HassOS. You cannot use these for HassOS.

Ok this part may be moved somewhere else